addons/gut/gut.gd

```diff
@@ -320,10 +321,22 @@ func _on_log_level_changed(value):
 # This is the most number of parameters GUT supports on signals.  The comment
 # on _on_watched_signal explains reasoning.
 # ------------------------------------------------------------------------------
-func _yielding_callback(from_obj=false,
+func _yielding_callback(
 		__arg1=null, __arg2=null, __arg3=null,
 		__arg4=null, __arg5=null, __arg6=null,
-		__arg7=null, __arg8=null, __arg9=null):
+		__arg7=null, __arg8=null, __arg9=null,
+		# one extra for the sentinel
+		__arg10=null):
+	var args = [
+		__arg1, __arg2, __arg3, __arg4, __arg5,
+		__arg6, __arg7, __arg8, __arg9, __arg10
+	]
+	var from_obj = false
+	for arg in args:
+		if arg == YIELD_FROM_OBJ:
+			from_obj = true
+			break
```

Comment:
Change this to:
```
var from_obj = args.contains(YIELD_FROM_OBJ)
```

Reply:
:facepalm: 

I used `arg in args`, 3.x doesn't seem to have `[].contains(value)`, it has `[].has(value)`